### DISCUSSION TITLE
Need help with a Panel-alike component

### DISCUSSION THREAD

**Discussion by BobanSpasic** [bgrabitmap/bgracontrols#185] [Q&A] [answered]
I am working on a Panel-alike component, but I realized that I miss some knowledge here.
How do I make this component to be the parent of the other components on it?
Something like TPanel or TGroupBox is?

![BCLeaBoard](https://github.com/bgrabitmap/bgracontrols/assets/68187526/298982af-7612-4f61-80cd-a015ca6ae6c1)

**Comment by BobanSpasic** [comment, marked as answer]
Got it, it needs csAcceptsControls in ControlStyle

**Comment by circular17** [comment]
Ok, that's good to know. I would have derived `TPanel`.
